package/MDAnalysis/analysis/distances.py

```diff
@@ -55,6 +55,8 @@
 
 import warnings
 import logging
+from .base import AnalysisBase
```

Comment:
It might be cleaner to create a new file for this analysis class.  It feels a little weird to import `base` into `distances`.  Once you get circular imports things get confusing fast.